build_runner/lib/src/entrypoint/options.dart

```diff
@@ -56,6 +57,24 @@ class BuildCommandRunner extends CommandRunner<int> {
       .join('\n');
 }
 
+/// Returns a map of output directory to root input directory to be used
+/// for merging.
+///
+/// Each output option is split on `:` where the first value is the
+/// output directory and the second value is the root input directory.
```

Comment:
In my original proposal it was `<root-level-dir>:<output>`, you have `<output>:<root-level-dir>`

@kevmoo doesn't feel strongly but his first instinct matched mine.

@jakemac53 - what is your opinion?

Reply:
Oh I made the exact same comment above - I agree with you it should be `<input-dir>:<output-dir>`

Reply:
Discussed with @jakemac53 and was able to convince him the current set up is better. He also brought up a good point that this aligns with the `port` syntax.

Reply:
(maybe?) final conversation about this we flipped again. The thing that convinced Jake and Kevin was looking at:

`pub run build_runner serve web:1234 --output web:deploy`

Using `<input-dir>:<output-dir>` has two benefits:
- Consistent placement of `web` before the colon since it means the same thing in both places.
- Matches the way we'd phrase this in english: "Sever the web directory on port 1234 and output the web directory to deploy".